cgaal-cli/Cargo.toml

```
@@ -1,15 +1,13 @@
 [package]
 name = "cgaal"
-version = "1.0.1"
+version = "1.1.0"
```

If adhering to SEMVER, because args are now not backwards compatible anymore (relevant for e.g. reproducibility pkgs) we should instead do:
```suggestion
version = "2.0.0"
```
Because:
> Given a version number MAJOR.MINOR.PATCH, increment the:

>    MAJOR version when you make incompatible API changes
>    MINOR version when you add functionality in a backward compatible manner
>    PATCH version when you make backward compatible bug fixes

>Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

---

True. Already version 2.0.0 🎉 ... We may have released 1.0.0 too soon

---

It's just an API number, measuring progress of CGAAL is closer tied to version of `cgaal-engine`, I'd say :shipit: